doc/examples/reconst_fwdti.py

```diff
@@ -0,0 +1,185 @@
+"""
+==========================================================================
+Using the free water elimination model to remove free water contamination
+==========================================================================
+
+As shown previously (see :ref:`example_reconst_dti`), the diffusion tensor
+model is a simple way to characterize the diffusion anisotropy. However, this
+model is not specific to particular type of tissue. For example, diffusion
+anisotropy in regions near the cerebral ventricle and parenchyma can be
+underestimated by partial volume effects of the cerebral spinal fluid (CSF).
+This free water contamination can particularly corrupt diffusion tensor imaging
+analysis of microstructural changes when different groups of subject show
+different brain morphology (e.g. brain ventricle enlargement associated with
+brain tissue atrophy that occurs in several brain pathologies and ageing).
+
+A way to remove this free water influences is to expand the DTI model to take
+into account an extra compartment representing the contributions of free water
+diffusion. The expression of the expanded DTI model is shown below:
+
+.. math::
+
+    S(\mathbf{g}, b) = S_0(1-f)e^{-b\mathbf{g}^T \mathbf{D}
+    \mathbf{g}}+S_0fe^{-b D_{iso}}
+
+where $\mathbf{g}$ and $b$ are diffusion grandient direction and weighted
+(more information see :ref:`example_reconst_dti`), $S(\mathbf{g}, b)$ is the
+diffusion-weighted signal measured, $S_0$ is the signal conducted in a
+measurement with no diffusion weighting, $\mathbf{D}$ is the diffusion tensor,
+$f$ the volume fraction of the free water component, and $D_iso$ is the
+isotropic value of the free water diffusion (normally set to $3.0 \times
+10^{-3} mm^{2}s^{-1}$).
+
+In this example, we show how to process a diffusion weighted dataset using the
+free water elimantion.
+
+Let's start by importing the relevant modules:
+"""
+
+import numpy as np
+import dipy.reconst.fwdti as fwdti
+import dipy.reconst.dti as dti
+import matplotlib.pyplot as plt
+from dipy.data import fetch_cenir_multib
+from dipy.data import read_cenir_multib
+from dipy.segment.mask import median_otsu
+
+"""
+Without spatial constrains the free water elimination model cannot be solved
+in data acquired from one non-zero b-value _[Hoy2014]. Therefore, here we
+download a dataset that was required from multi b-values.
+"""
+
+fetch_cenir_multib(with_raw=False)
+
+"""
+From the downloaded data, we read only the data acquired with b-values up to
+2000 $s.mm^{-2} to decrease the influence of non-Gaussain _[Hoy2014].
```

Comment:
"non-Gaussain" => "non-Gaussian"

Also, non-Gaussian what? This sentence seems to cut off mid-way